nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -30,6 +30,8 @@
 
 - `mariadb` now defaults to `mariadb_1011` instead of `mariadb_106`, meaning the default version was upgraded from 10.6.x to 10.11.x. See the [upgrade notes](https://mariadb.com/kb/en/upgrading-from-mariadb-10-6-to-mariadb-10-11/) for potential issues.
 
+-  The `matrix-synapse` package does not accept the `enableSystemd` and `enableRedis` arguments anymore. All extras can now instead be configured from services.matrix-synapse.extras. Most of them will automatically be discovered and installed, when the relevant configuration sections are present.
```

Comment:
It would be nice to linkify `services.matrix-synapse.extras`. Also, it's probably reasonable to explain a few more implications for those who don't use the module at all (i.e. that `extras` is supposed to be used and that `matrix-synapse` vs `matrix-synapse-unwrapped` are different things).

Reply:
Expanded, I hope the sentence structure still makes sense now,